### DISCUSSION TITLE
How to check/test that containarized nginx is running properly?

### DISCUSSION THREAD

**Discussion by IavTavares** [nginx/kubernetes-ingress#2571] [Q&A] [answered]
I've dockerized  a nginx+gunicorn+flask app. 

I was wondering how we could tell whether the container is running smoothly as expected, and specially that nginx is doing what's supposed to do (e.g.load balancing, etc.)...

So, far I've run a few curls commands, and they have been successful. However, they were also successful before the nginx addition. I would like to test nginx specific capabilities.

I'm completely new to this. Thanks for all the help.

**Comment by brianehlert** [comment, marked as answer]
If you are using open source NGINX you will probably find the stub status module useful for simple troubleshooting:  https://nginx.org/en/docs/http/ngx_http_stub_status_module.html

**Reply by IavTavares** [reply]
@brianehlert  Thanks.
